Assets/_Project/Code/Models/teamClass.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System.Collections.Generic;
+using System.Security.AccessControl;
+using UnityEngine;
+
+using System.Text;
+
+public class Team
+{
+    //zadeklarowanie nazwa zmiennych
+    private string name;
+    private string colour;
+    private int money= 10000;
+    private int cluesUsed=0;
+    private int inactiveRounds = 0; //licznik rund bierności w licytacji
+    private int bid = 0;
+    private ulong id;
+    //powerupy zmienione z listy stringów na inta. Po prostu będziemy mieć listę powerupów gdzieśtam w kodzie wpisaną, a żeby sprawdzić
+    //czy drużyna ma danego powerupa trzeba bd zrobić 
+    /*
+      if( (powerUps&powerup)!=0)){
+    do thing
+    }
+     */ 
+    //działa tak samo a mniej zabawy z serializacją listy w liście x.x
+    //no i zamiast lecieć przez listę mamy jednego ifa
+    private int powerUps = 0;
+    //możliwość odczytania danych
+    public string Names => name;
+    public int Money => money;
+    public int CluesUsed => cluesUsed;
+    public int InactiveRounds => inactiveRounds;
+    public int PowerUps => powerUps;
+    public string Colour => colour;
+    public int Bid => bid;
+    public ulong ID => id;
+    //dodawanie pieniędzy
+    public void AddMoney(int balance)
+    {
+        if (balance >= 0)
+        {
+            money += balance; //dodawanie pieniędzy
+        }
+        else
+
+        {
+            
+            Debug.LogWarning("Nie można dodać ujemnej kwotry."); //wyświetlanie informacji że nie można dodać ujemnej kwoty
+        }
+    }
+    //odejmowanie pieniędzy
+    public void SubstractMoney(int balance)
+    {
+        if (balance >= 0)
+        {
+            money -= balance; //odjemowanie pieniędzy
+            if (money <= 0)
+            {
+                money = 0; //jeżeli pieniądze po odjęciu zejdą poniżej zera ustawiam pieniądzeNaKoncie na 0
+            }
+        }
+        else
+        {
+            Debug.LogWarning("Nie można odjąć ujemnej kwoty."); //wyświetlanie informacji że nie można odjąć ujemnej kwoty
+        }
+    }
```

Comment:
zamiast tworzyć metody add i remove etc. użyj [operatorów](https://stackoverflow.com/questions/32183568/how-to-make-a-custom-add-method-for-classes-in-c-sharp)